src/runtime/rpc/rpc_endpoint.cc

```diff
@@ -1042,12 +1069,38 @@ class RPCClientSession : public RPCSession, public DeviceAPI {
   bool IsLocalSession() const final { return false; }
 
  private:
+  void RPCMaxTransferRemoteReturnValue(TVMArgs args) {
+    // Use args[1] as return value, args[0] is tcode
+    rpc_chunk_max_size_bytes_ = (uint64_t)args[1];
+  }
+
+  uint64_t GetRPCMaxTransferSize() {
+    PackedFuncHandle rpc_func = GetFunction("tvm.rpc.server.GetCRTMaxPacketSize");
```

Comment:
here you could also just do this once, too

Reply:
done